gsl/span

```diff
@@ -347,6 +347,7 @@ class span
 public:
     // constants and types
     using element_type = ElementType;
+    using value_type = ElementType;    
```

Comment:
I'm not so sure about this. First, adding `std::remove_cv_t` causes a lot of compilation errors, than I look how `value_type` is implemented in `vector` - it's directly type of the element (cv is not removed).